src/pages/docs/deployments/custom-scripts/run-a-script-step.md

```diff
@@ -56,6 +56,12 @@ You may also select the source of the script, either:
 Using scripts from inside a package or a git repository are a great way to version and source control your scripts. (You can be assured the correct version of your script will be run when deploying each version of your application.) Both methods have benefits and suit different applications: choose the method best suited to your situation.
 :::
 
+:::div{.hint}
+From Octopus **2024.1**, if you are using [version control](/docs/projects/version-control) with your project you can source files from the same Git repository as your deployment process by selecting Project as the Git repository source. When creating a Release, the commit hash used for your deployment process will also be used to source the files.
+
+You can find more information about this feature in [this blog post on using Git resources directly in deployments](https://octopus.com/blog/git-resources-in-deployments).
```

Comment:
A nit would be to exclude the `this` from the link text. Not helpful for someone scanning links with a screen reader.

Reply:
Ah, I added in a fuller description to the link to try and make sure it catered well for screen readers, didn't realise the `this` was unhelpful, thanks for the heads up.

Reply:
And for that I am grateful!